incubator/nfs-pv-provider/templates/nfs-server-pv.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "{{ .Values.storage.name }}"
+  labels:
+    app: {{ template "nfs-pv-provider.name" . }}
+    chart: {{ template "nfs-pv-provider.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    demo: "{{ .Values.storage.demo_label }}"
+spec:
+  accessModes: [ "ReadWriteOnce" ]
+  storageClassName: "{{ .Values.storage.storageClass }}"
```

Comment:
Use our best practice for storage class cofiguration: #1869

Reply:
I think I understand what you mean, do it like this: https://github.com/bryanlarsen/charts/blob/5d520eee1c62c690d0d8c77bd8ab20ea99f17c0e/stable/prometheus/templates/alertmanager-pvc.yaml

Implemented in b8339cd4